### ISSUE TITLE
Feature request: Diana helper

### ISSUE THREAD

**Issue by Fluboxer** [SkyblockerMod/Skyblocker#240]
Diana events happen once in a while and it would be nice to have functionality to help with them

Things I'm looking forward are: 
1. Find approximate burrow location based on spade particles or arrow from previous burrow. Draw waypoint on approx location
2. Detect near burrows. Draw waypoint on those  
3. Message about inquisitor spawn in chat (should support picking normal chat/party chat). Should be parseable by 1.8 popular mods with such functionality if possible
4. Ability to parse said messages and place a waypoint to them. Should support 1.8 popular mod formats 
5. Warp to hub warps on a button if it gets you closer to burrow/inquisitor waypoint 
6. Fix arrows pointing upwards (if possible; hypixel multiversion support moment)
7. Hit counter on Gaia; Box vulnerable cat (if possible)

**Comment by Julienraptor01** [comment]
The feature is present in SkyblockMod-Fabric iirc

Also for 6, there's a solution...
Going slower...
Thanks hypixel for the multiversion
Basically hypixel only correct the arrow facing after like 2 secs

**Comment by Julienraptor01** [comment]
> I tried this mod and while their github page didn't mention it, they do indeed have some Diana functionality. It doesn't cover 1 _(or it refuses to do so due to some bug? is it supposed to triangulate? hard to tell as it literally named "Mythological Ritual helper")_ but it does cover 2 - although their waypoints lack persistence (if you take off your shovel, they vanish too), uses same color for both new burrow and current chain burrow (possible confusion) and green waypoint in green place looks a bit too camouflaged. It still helped me a lot, so thank you for mentioning that!
> 
> It kinda brings up a question that were bugging me for years - do I have to install separate mod, disable 99.95% of it's functionality for single feature because **big mod** that already satisfies 99% of my needs happen to lack that one particular thing, praying that I didn't miss any other feature and it wouldn't mess anything up? It was a problem on 1.8 and it would be sad to see 1.20 make same mistake. On top of that they have shortcuts you can't disable, meaning that you can accidentally send yourself in Brazil
> 
> btw I guess Hypixel doesn't like me as arrows refuse to do the thing ![изображение](https://user-images.githubusercontent.com/36457056/260548175-89da1798-0abc-42ab-a80d-b5198e4f4368.png)

this other mod is currently merging in this one btw, but it's not completly done...

**Comment by kevinthegreat1** [comment]
I am the developer for Skyblock Mod. I was developing some Diana features and basically got frustrated and stopped working on it. So a minimal version was shipped. I also develop this mod and working to merge the two. Now I’m working on dungeons secrets and will take a look at Diana once that is done.

**Comment by viciscat** [comment]
Kevin added some Diana things in his mod recently so uh check that out
